kompendium-auth/src/main/kotlin/org/leafygreens/kompendium/auth/KompendiumAuth.kt

```diff
@@ -0,0 +1,51 @@
+package org.leafygreens.kompendium.auth
+
+import io.ktor.auth.Authentication
+import io.ktor.auth.basic
+import io.ktor.auth.BasicAuthenticationProvider
+import io.ktor.auth.jwt.jwt
+import io.ktor.auth.jwt.JWTAuthenticationProvider
+import org.leafygreens.kompendium.Kompendium
+import org.leafygreens.kompendium.models.oas.OpenApiSpecSchemaSecurity
+
+object KompendiumAuth {
+
+  init {
+      Kompendium.pathCalculator = AuthPathCalculator()
+  }
+
+  fun Authentication.Configuration.notarizedBasic(
+    name: String? = null,
+    configure: BasicAuthenticationProvider.Configuration.() -> Unit
+  ) {
+    Kompendium.openApiSpec.components.securitySchemes[name ?: "default"] = OpenApiSpecSchemaSecurity(
+      type = "http",
+      scheme = "basic"
+    )
+    basic(name, configure)
+  }
+
+  // TODO move jwt to separate module?
```

Comment:
let's remove this todo, I think for v1 having auth as a single library makes the most sense

Reply:
check, it is removed